build-system/server/new-server/router.js

```diff
@@ -15,6 +15,7 @@
  */
 
 const router = require('express').Router();
+// @ts-ignore
```

Comment:
Totally good with this. The stuff in `dist/transform` is output that we shouldn't be type-checking. (I assume adding this directory to the ignore list isn't sufficient?)

Reply:
Adding it to the `tsconfig.excludes` does not stop it from being checked as a dependency. This is here to handle the situation where `new-server` hasn't been built yet and thus `dist` does not exist